### ISSUE TITLE
Needs test

### ISSUE THREAD

**Issue by Lendude** [Realityloop/markup#4]
Some basic test coverage would be great.

**Comment by Decipher** [comment]
Absolutely, but as this is a port I am not wishing to add functionality at this stage. Where tests present in the D7 release I would have ported them first to ensure all functionality worked correctly in D8 as it did in D7, but that is not the case.

I would fully intend for tests to be added once the code has been added to Drupal.org.
